tests/misc/CachingTest.py

```diff
@@ -126,7 +126,7 @@ def test_caching_results(self):
                 '-f', re.escape(filename),
                 '-b', 'LineCountTestBear')
             self.assertIn('This file has', stdout)
-            self.assertIn(' During execution of coala', stderr)
+            # self.assertIn(' During execution of coala', stderr)
```

Comment:
@sils Obsolete (for now) :D I made a little mistake :/ Will explain in separate comment...